src/Grammar.ts

```diff
@@ -5,11 +5,13 @@ import { Context } from './Context';
  
 const endTagRe = /<\/?(emu-\w+|h?\d|p|ul|table|pre|code)\b[^>]*>/i;
 const globalEndTagRe = /<\/?(emu-\w+|h?\d|p|ul|table|pre|code)\b[^>]*>/ig;
-const entityRe = /&(gt|lt|amp);/g;
+const entityRe = /&(gt|lt|amp|(l|r)dquo);/g;
 const entities: { [key: string]: string } = {
   "&gt;": ">",
   "&lt;": "<",
-  "&amp;": "&"
+  "&amp;": "&",
+  "&ldquo;": "“",
+  "&rdquo;": "”",
```

Comment:
How about using `"\u201C"` and `"\u201D"` so the source code remains ASCII-safe?